homeassistant/components/izone/climate.py

```diff
@@ -0,0 +1,468 @@
+"""
+Support for the iZone HVAC.
+
+For more details about this platform, please refer to the documentation at
+https://github.com/Swamp-Ig/izone_custom_component
+"""
+import logging
+from typing import cast
+
+from pizone import Zone, Controller
+
+from ..climate import ClimateDevice
```

Comment:
I think we should use

```suggestion
from homeassistant.components.climate import ClimateDevice
```

instead (relative imports only for in-integration imports)